support-frontend/app/controllers/DigitalSubscriptionController.scala

```diff
@@ -50,6 +50,8 @@ class DigitalSubscriptionController(
   def digital(countryCode: String, orderIsAGift: Boolean): Action[AnyContent] = {
     MaybeAuthenticatedAction { implicit request =>
       implicit val settings: AllSettings = settingsProvider.getAllSettings()
+      val defaultPromos = priceSummaryServiceProvider.forUser(isTestUser = false).getDefaultPromoCodes(DigitalPack)
```

Comment:
Should we be using the [test user value](https://github.com/guardian/support-frontend/pull/5456/files#diff-73774672ba93580adbffcdbbd2cbf93499940fa5b20598b4dec1949dab6b786bR68) here?